content/english/android-tictactoe/booleans.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Booleans"
+date: 2020-07-27T14:17:07.42-07:00
+draft: true
+weight: 9
+---
+
+# Booleans
+
+Booleans are true or false statements. Unlike strings or numbers, booleans store statements of truth: is what I am saying true or false? For example, if I ask, "Are you a robot?", this question produces a true or false result, which we call a Boolean. In this case, since you are not a robot (hopefully!), we would produce false. We can also use math operators to create boolean expressions. Here are some examples; however, notice the unusual symbols for "equal to" and "not equal to":
+< Less than > Greater than
+<= Less than or equal to => Greater than or equal to
+== Equal to != Not euqal to
+As usual, use `System.out.println` to print out your results:
+
+```java
+System.out.println(10 < 8);
+System.out.println((3 * 6) == (32 - 14));
+```
+{{% notice tip %}}
+## Working Together
+
+Delete all your `System.out.println` text in your code. Try guessing the answers to the following expressions. Use `System.out.println` to check your answers.
+
+- 54 < (10 + 32)
+- (37 / 5) == 7
+- "Hello" + "World" == "Hello World"
+- false == false
+{{% /notice %}}
+
+## Booleans Operators
+
+You can also connect boolean expressions together using the `&&` (AND) and the `||` (OR) operator. For example, suppose I ask: "Are you a human, and is Nuvi a robot?" The word "and" connects the two true-false questions together. In this case, since it is true that you are a human, and it is also true that Nuvi is a robot, then the overall result is `true`. Here's a chart that describes what happens when we connect booleans together:  
+<table class="table">
```

Comment:
Same as above, use Markdown to define the table (check out some of the other workshops on how this can be done such as the C# workshop).

Reply:
I initially tried Markdown, including copying it directly from the C# workshop, but it isn't displaying correctly in my Hugo dev environment. I will switch these tables over to Markdown and assume it's a quirk with my Hugo setup.